DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateProgress/EditCompletionDate.cshtml

```diff
@@ -0,0 +1,57 @@
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewComponents
+@using DigitalLearningSolutions.Web.Views.TrackingSystem.Delegates.DelegateProgress
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@model EditCompletionDateViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Edit Completion Date" : "Edit Completion Date";
+
+  var routeParamsForBackLink = new Dictionary<string, string?>();
+
+  if (Model.AccessedVia.Equals(DelegateProgressAccessRoute.CourseDelegates)) {
+    routeParamsForBackLink.Add("accessedVia", Model.AccessedVia.ToString());
+    routeParamsForBackLink.Add("progressId", Model.ProgressId.ToString());
+  } else {
+    routeParamsForBackLink.Add("delegateId", Model.DelegateId.ToString());
+  }
+  var exampleDate = DateTime.Today;
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@new []{ nameof(@Model.Day), nameof(@Model.Month), nameof(@Model.Year)}" />
+    }
+
+    <h1 class="nhsuk-heading-xl">Edit completion date for @Model.CourseName</h1>
+
+    <p class="nhsuk-lede-text nhsuk-u-margin-top-4">
+      For example 1 7 2021. Leave the boxes blank to clear the completion date.
+    </p>
```

Comment:
Do we really need this here? We have the same hint text further down.

Reply:
got rid